scss/modules/_image-focus.scss

```diff
@@ -14,7 +14,7 @@
 
 	&__clickarea {
 		position: absolute;
-		z-index: 99;
+		z-index: 100;
```

Comment:
I have updated this in the following feature: feature/improve-code-and-functionality